### PR TITLE
week19_complete

### PR DESCRIPTION
**Pull request by anthonyman15**
Completed week 19 lab sheet
New test methods

### CODE REVIEW

File: AdminSystem/Web.config [hunk 1]
```diff
@@ -5,12 +5,12 @@
```
Comment (Dylan773) [inline AdminSystem/Web.config:5]: Framework change?

File: ClassLibrary/clsStock.cs [hunk 2]
```diff
@@ -1,32 +1,109 @@
...
     public class clsStock
     {
-        public clsStock AStock = new clsStock();
-        public int ProductId;
-        public string ProductName;
-        public string ProductDescription;
-        public Boolean IsAvailable = false;
-        public int QuantityAvailable = 0;
-        public DateTime RestockDate;
+        private int aProductId;
```
Comment (Ashley-exe) [inline ClassLibrary/clsStock.cs:7]: Fairly sure these privates were supposed to be prefaced with `p` not `a`

File: ClassLibrary/clsStock.cs [hunk 3]
```diff
@@ -1,32 +1,109 @@
...
         {
-            ProductId = 1;
-            ProductName = "Beef";
-            ProductDescription = "About Something...";
-            IsAvailable = true;
-            QuantityAvailable = 10;
-            RestockDate = DateTime.Now;
-
-            return true;
+            clsDataConnection DB = new clsDataConnection();
```
Comment (Ashley-exe) [inline ClassLibrary/clsStock.cs]: We shouldn't need to create this everytime, we can make this static.

File: TestingStaff/UnitTest1.cs [hunk 4]
```diff
@@ -1,5 +1,6 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
```
Comment (Ashley-exe) [inline TestingStaff/UnitTest1.cs:1]: Is this file needed? it's empty, and doesn't even include the testing libraries

File: TestingStaff/tstStock.cs [hunk 5]
```diff
@@ -47,11 +65,107 @@ public void AttributeQuantityAvailableOK()
...
+
+        [TestMethod]
+        public void TestProductNameFound()
+        {
+            clsStock product = new clsStock();
+            Boolean Found = false;
+            Boolean OK = true;
+            Int32 ProductId = 2;
+            Found = product.Find(ProductId);
+            if(product.ProductName != "Product Name")
```
Comment (Ashley-exe) [inline TestingStaff/tstStock.cs:100]: These test shouldn't use custom booleans, instead the test should be along the lines of 
`Assert.IsTrue(product.ProductName == "Product Name")`

**Review by Ashley-exe — CHANGES_REQUESTED** [review]
Mostly looks good, just a could things to cover
- [x] I'll add a static database reference
- [x] tests need to use `Assert` for tests, instead of using if statements. (Ideally, but the lab sheets are full of shit practice so)
- [x] There's a blank, unused test file that should be removed.

**Comment by Ashley-exe** [comment]
Hold up, @fairywizard15 , the tests are done with bad practice but is that the way the lab sheets say to do them?

if so, fuck it. It's not good, but if they ask for it, they get it.

**Review by Ashley-exe — APPROVED** [review]
Close enough

**Review by Dylan773 — APPROVED** [review]
Good shit

**Pull request merged**